src/sass/common.scss

```diff
@@ -122,6 +122,36 @@ label {
     }
 }
 
+.form-validation-message h4 {
+    font-size: 13px !important;
+    font-style: italic !important;
+    color: $COLOR_BLACK !important;
+    background-color: transparent !important;
+    background-image: none !important;
+    border: 0 !important;
+    padding: 5px 0px 10px !important;
+}
+
+.form-validation-message h4:after {
```

Comment:
this can be written under form-validation-message h4 as sass styling `&:after` on line 132 instead of a separate section since they have the same parent